volume/ceph/ceph.go

```diff
@@ -15,7 +17,11 @@ import (
 	"github.com/opencontainers/runc/libcontainer/utils"
 )
 
-const CephImageSizeMB = 1024 * 1024 // 1TB
+const (
+	CephImageSizeMB   = 1024 * 1024 // 1TB
+	LuksDevMapperPath = "/dev/mapper/"
+	cryptoLUKS        = "crypto_LUKS"
```

Comment:
Fixed in https://github.com/medallia/docker/pull/15/commits/975c9c2ab2e543bd69892b3254de4e3e90c1156e